using-valtimo/dashboard/create-dashboard.md

```diff
@@ -0,0 +1,23 @@
+# Creating a dashboard
```

Comment:
The way that this description is set up is less extensive than most other using valtimo pages. Most use screenshots to illustrate the flow as well.